setup.cfg

```diff
@@ -3,21 +3,23 @@
 # https://flake8.readthedocs.io/en/latest/user/configuration.html
 # https://flake8.readthedocs.io/en/latest/user/error-codes.html
 
-# Aligned with black https://github.com/psf/black/blob/main/.flake8
-extend-ignore = E203, E266, E501
 # Note: there cannot be spaces after commas here
 exclude = __init__.py,versioneer.py,distributed/_concurrent_futures_thread.py
 ignore =
-    E4,         # Import formatting
-    E731,       # Assigning lambda expression
-    W503,       # line break before binary operator
-
+    # Ignores below are aligned with black https://github.com/psf/black/blob/main/.flake8
+    E203, # Whitespace before ':'
+    E266, # Too many leading '#' for block comment
+    E501, # Line too long
```

Comment:
Black ignores this because they use a different error code for line too long, we shouldn't ignore it